src/pages/welcome/index.js

```diff
@@ -8,15 +8,17 @@ import "./style.css";
 const Welcome = () => {
   const { onCreateProfile } = useAuth();
 
+  const [profile, setProfile] = useState({
+    firstName: "",
+    lastName: "",
+    githubUsername: "",
+    bio: "",
+    password: "",
```

Comment:
Maybe swap password for something more contact info related? Password is already assigned and verified when they sign up.

Reply:
Thought that the field is being used for the profile page and will be editable there which team 4 is handling